mercury_app/utils.py

```diff
@@ -18,15 +18,15 @@ def get_api_organization(token):
     Get organization from the user of the token from the api
```

Comment:
Shouldn't the name of this method be `get_api_organizationS`, please note that the `'organizations'` key is a list.